src/briefcase/platforms/windows/__init__.py

```diff
@@ -285,6 +276,22 @@ def package_app(
             )
             self.sign_file(app=app, filepath=self.binary_path(app), **sign_options)
 
+        if app.packaging_format == "zip":
+            self._package_zip(app)
+        else:
+            self._package_msi(app)
+
+        if sign_app:
+            self.logger.info("Signing package...", prefix=app.app_name)
+            self.sign_file(
+                app=app,
+                filepath=self.distribution_path(app),
+                **sign_options,
+            )
```

Comment:
This will attempt to sign the .zip file, which won't work because .zip files don't allow for signing.

Reply:
Yes, I asked this in the conversation above, but then I didn't wait for your answer. Sorry. Google search results have let me believe that this is possible. Seems that it is not possible with Windows' SDK SignTool.

Reply:
This looks like the only outstanding item.